pkgs/dapNode/js-debug/README.md

```diff
@@ -0,0 +1,14 @@
+# js-debug Package
+
+This Nix package installs and builds the DAP server in
+https://github.com/microsoft/vscode-js-debug.
+
+The `.nix` files were generated using [node2nix](https://github.com/svanderburg/node2nix) by the command:
+
+```
+node2nix -d
+```
+
+which uses the `package.json` file as input, which was grabbed from https://github.com/svanderburg/node2nix
+at rev 483425e50cf0fb313610d2346b51459c8db556e2, then removing the playwright package as I couldn't
```

Comment:
is the rev for node2nix or for vscode-js-debug? if it's for the former, isn't that the version that's pinned in this repo? if not, can it be added to the devshell?